src/oaklib/utilities/mapping/sssom_utils.py

```diff
@@ -74,14 +73,7 @@ def emit(self, obj: Mapping):
         self.mappings.append(obj)
 
     def finish(self):
-        mset = MappingSet(
-            mapping_set_id="temp", mappings=self.mappings, license=UNSPECIFIED_MAPPING_SET_ID
-        )
-        if self.ontology_interface:
-            prefix_map = self.ontology_interface.prefix_map()
-        else:
-            prefix_map = {}
-        doc = MappingSetDocument(prefix_map=prefix_map, mapping_set=mset)
-        msdf = to_mapping_set_dataframe(doc)
+        converter = self.ontology_interface.converter if self.ontology_interface else None
+        msdf = MappingSetDataFrame.from_mappings(self.mappings, converter=converter)
```

Comment:
default metadata is applied inside this function, so it doesn't need to be done directly